tests/yelp_bytes_test.py

```diff
@@ -230,3 +230,27 @@ def test_to_bytes_is_like_str_encode(value):
         to_bytes_result = '(error)'
 
     assert bytes_result == to_bytes_result
+
+
+@pytest.mark.parametrize('value', (
+    UNICODE.ascii,
+    UNICODE.latin1,
+    UNICODE.win1252,
+    UNICODE.bmp,
+    UNICODE.utf8,
+    u"this is a unicode string",
+    str("native string")
+))
```

Comment:
You've only walked through cases of unicode with one ascii bytestring (for py2).

These need to be pumped up.

Throw in the "able" classes above, too.